app/src/main/java/net/skyscanner/backpack/demo/data/ComponentRegistry.kt

```diff
@@ -86,14 +90,23 @@ class NodeItem(
     var fullName = this.name
 
     while (parent != null) {
-      fullName = "${parent.name} - $name"
+      fullName = "${parent.name} - $fullName"
```

Comment:
this was a bug if there were more than 2 node levels

Reply:
What was the bug?

Reply:
It would skip some levels when assembling the full name - e.g. if the level is Button - Compose - Primary it would end up with Button - Primary as it would just use the name, rather than the partial full name (as in Primary instead of Compose - Primary)

Reply:
Do we need to update the website screenshots taking code to reflect that change?

Reply:
No - the `DocsRegistry` takes a list of ids (which is the full names), so we define the full name and then starts the details fragment. The correct item is then resolved based on that id using the same code as here. 
We currently don't have any items that have more than 2 levels, so we never ran into this issue. However with tabs this is much more likely - e.g. say we add a button component for compose we'll have button -> view -> all the different options (and probably also for the compose version)